readme/product.md

```diff
@@ -2,9 +2,18 @@
 
 The `product` subfolder allows to import products provided as CSV files. See the example below:
 
-| id | name | type | product_variant_ids/categ_id/id 
-| - | - | - | - | - | - | - | - | - | - | - | - |
-| init.testProduct | My Product | service | product.product_category_all
+| id | product_variant_ids/id | 
+| - | - | 
+| init.testProduct | init.testProduct
 
 See the test resources for further examples:
 - [odoo_csv/product/product.template.csv](../odoo_initializer/tests/resources/odoo_csv/product/product.template.csv)
+
+
+:exclamation:
+**Note:** 
+Product model which is represeneted by `product.template` Odoo Model is supposed to represent a product definition in which it could have multiple variation (`product.product`). 
```

Comment:
**Note:** Product vs Product Variant 
The `product` domain is represented by `product.template` model. It can be used as a template for its variations, called `product variants` (`product.product`).

To define a product variant please use [Product Variant](product_variant.md) domain instead.

This domain is the place where to create the mappings between the product and its variants as shown above in the example.